pkgs/applications/office/timetable/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchFromGitHub
+, glib
+, gtk3
+, hicolor-icon-theme
+, json-glib
+, libgee
+, meson
+, ninja
+, pkgconfig
+, pantheon
+, python3
+, wrapGAppsHook
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "timetable";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "lainsce";
+    repo = "timetable";
```

Comment:
```suggestion
    repo = pname;
```

Last one :smile:

Reply:
well, I shouldn't have miss this one :disappointed: